src/fingerprint.rs

```diff
@@ -262,7 +263,19 @@ fn remove_not_built_with_in_a_profile(
     total_disk_space += remove_not_matching_in_a_dir(&dir.join("deps"), &keep, dry_run)?;
     // examples is just final artifacts not tracked by fingerprint so skip that one.
     // incremental is not tracked by fingerprint so skip that one.
-    total_disk_space += remove_not_matching_in_a_dir(&dir.join("native"), &keep, dry_run)?;
+    // native is no longer used by cargo as of August 2019
+    match remove_not_matching_in_a_dir(&dir.join("native"), &keep, dry_run) {
+        Ok(bytes) => total_disk_space += bytes,
+        Err(err) => {
+            if let Some(ioerr) = err.downcast_ref::<io::Error>() {
+                if ioerr.kind() != io::ErrorKind::NotFound {
+                    return Err(err);
```

Comment:
Will this not result in no total disk space being reported?
I would perfer that we check if the directory exists, using `Path.exists()` and logging some info message using the logger if this is not the case.